MonstieWash/Assets/Scripts/UIManager.cs

```diff
@@ -36,19 +50,105 @@ private void ToggleUIVisibility()
         CBAnimator.SetBool("Hide", m_UIVisible);
     }
 
-    // Update is called once per frame
-    private void Update()
+/// <summary>
+/// Public method used by TaskTracker to initialise clipboard after all task trackers have been initialised. Creates filters for tasks to show on each scene.
+/// </summary>
+/// <param name="keys">A list of all the keys used to identify tasks tracked in TaskTracker</param>
+    public void LoadKeys(List<string> keys)
+    {
+        m_TaskKeys = keys;
+        foreach (var scene in m_roomSaver.m_allScenes)
+        {
+            switch (scene)
+            {
+                case "Overview":
+                    m_SceneTasks.Add(scene, m_TaskKeys.FindAll(s => s.Contains("Overall") && s.Count(x => x == '#') <= 1));
+                    break;
+                default:
+                    var sceneKeys = m_TaskKeys.FindAll(s => s.Contains(scene));
+                    sceneKeys.Add("Overall");
+                    m_SceneTasks.Add(scene, sceneKeys);
+                    break;
+            } 
+        }
+        InitialiseClipboard(m_TaskContainer, "", 0);
+        UpdateClipboardUI("Overview");
+    }
+
+/// <summary>
+/// Recursively creates objects inside the clipboard object for each layer of tasks across all monster scenes. 
+/// </summary>
+/// <param name="currentParent">The parent object of the layer of tasks current being initialised.</param>
+/// <param name="currentParentName">The name identifier of the current parent object.</param>
+/// <param name="taskLayer">The current depth of the layer of tasks being processed.</param>
+    private void InitialiseClipboard(GameObject currentParent, string currentParentName, int taskLayer)
+    {
+        var currentTaskLayer = m_TaskKeys.FindAll(s => s.Contains(currentParentName) && s.Count(x => x == '#') == taskLayer);
+        foreach (var task in currentTaskLayer)
+        {
+            var newTaskObject = GameObject.Instantiate(m_TaskTextPrefab, currentParent.transform);
```

Comment:
Small, thing, just don't need to add the `GameObject.` for `Instantiate`.
```suggestion
            var newTaskObject = Instantiate(m_TaskTextPrefab, currentParent.transform);
```